test/functional/p2p_ibd_txrelay.py

```diff
@@ -5,9 +5,23 @@
 """Test fee filters during and after IBD."""
```

Comment:
Maybe update this, since the test is now testing more.

Reply:
Done!